source/guide_searx.rst

```diff
@@ -0,0 +1,242 @@
+
+----
+
+.. note:: The port with ``8888`` will not be touched, but keep this number in your mind for later configurtations.
+
+.. code-block:: yaml
+ :emphasize-lines: 5,13,14,15
+
+use_default_settings: True
+
+general:
+    debug : False # Debug mode, only for development
+    instance_name : "searx" # displayed name
+
+search:
+    safe_search : 0 # Filter results. 0: None, 1: Moderate, 2: Strict
+    autocomplete : "" # Existing autocomplete backends: "dbpedia", "duckduckgo", "google", "startpage", "swisscows", "qwant", "wikipedia" - leave blank to turn it off by default
+    default_lang : "" # Default search language - leave blank to detect from browser information or use codes from 'languages.py'
+
+server:
+    port : 8888
+    bind_address : "0.0.0.0" # address to listen on
+    secret_key : "01234567890123456" # change this!
+    base_url : False # Set custom base_url. Possible values: False or "https://your.custom.host/location/"
+    image_proxy : False # Proxying image results through searx
+
+# uncomment below section if you have running morty proxy
+#result_proxy:
+#    url : http://127.0.0.1:3000/
+#    key : !!binary "your_morty_proxy_key"
+
+
+Step 4 - Supervisord Setup
+--------------------------
+
+At first we must create the service file ``~/etc/services.d/searx.ini`` and the following content:
+
+.. warning:: The PATH variable consider the installed Python modules regarding to the version 3.6 at this time point. Please have a look in the directory ``~/.local/lib/`` to be sure which version is the actual ones.
+
+.. code-block::
+ :emphasize-lines: 2
+
+[program:searx]
+environment =
+ PATH="%(ENV_HOME)s/opt/searx/:%(ENV_HOME)s/.local/lib/python3.6/:%(ENV_PATH)s",
+ SEARX_SETTINGS_PATH="%(ENV_HOME)s/etc/searx/settings.yml"
+autostart=yes
+autorestart=yes
+command=python3 %(ENV_HOME)s/opt/searx/searx/webapp.py
+
+
+We must report to supervisord that there is a new ini file to consider:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ supervisorctl reread
+ searx: available
+
+
+ And then to start the daemon:
+
+ .. code-block:: console
+
+ [isabell@stardust ~]$ supervisorctl update
+ searx: added process group
+
+
+At this time point, searx should be run in the background.
+
+Step 5 - Domain Setup
+---------------------
+
+This guide consider a subdomain ``searx.isabell.example`` for your own searx instance.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ uberspace web domain add searx.isabell.example
+
+.. note:: Please be sure, that your new subdomain is configured at your domain provider too.
+
+If you want to find out that everthing is maintained for the web server on your Uberspace account, use the command:
+
+.. code-block:: console
+
+[isabell@stardust ~]$ uberspace web domain list
+isabell.example
+searx.isabell.example
+isabell.uber.space
+
+Step 6 - Web Backend Setup
+--------------------------
+
+This step is important, that your running searx instance is reachable from outside.
```

Comment:
Please use the [web backend include](https://github.com/Uberspace/lab/blob/main/source/includes/web-backend.rst) here, like in the [1up guide](https://lab.uberspace.de/guide_up1.html#configure-web-server).

Reply:
Ok.